packages/cli/templates/default-plugin/package.json.hbs

```diff
@@ -1,14 +1,15 @@
 {
-  "name": "@backstage/plugin-{{id}}",
+  "name": "{{#if scopeName}}{{scopeName}}/{{/if}}plugin-{{id}}",
   "version": "{{version}}",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
-  "private": true,
+  "private": {{privatePackage}},
   "publishConfig": {
     "access": "public",
     "main": "dist/index.esm.js",
-    "types": "dist/index.d.ts"
+    "types": "dist/index.d.ts"{{#if registryURL}},
+    "{{scopeName}}:registry":"{{registryURL}}"{{/if}}
```

Comment:
if you put this as the first line instead of the last, you don't have to go through the trouble of going across newlines - would be more readable

Reply:
noted.